README.md

```diff
@@ -1,9 +1,36 @@
-cat README.MD
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nick Lesho . Jarvis Consulting
 
+I graduated with a Computer Science and Economics degree from the University of Toronto in June of 2021. Upon graduating, I worked as part of a 3-month contract for Canadian Musicians Co-operative as a Computer Systems Analyst. My duties included web development, IT support, and debugging an Android application. Currently, I am a technical consultant at Jarvis. I am passionate about web development, with proficiency in both front-end and back-end web development. I am especially passionate about using technical skills to solve problems for a wide variety of stakeholders. With the tech industry constantly changing and evolving, I am excited about learning and using all the new technologies that will unfold.
```

Comment:
I graduated with a Computer Science and Economics degree from the University of Toronto in June 2021. Upon graduating, I worked as part of a 3-month contract for Canadian Musicians Co-operative as a Computer Systems Analyst. My duties included web development, IT support, and debugging an Android application. Currently, I am a technical consultant at Jarvis. I am passionate about web development, with proficiency in front-end and back-end web development. I am incredibly passionate about using technical skills to solve problems for a wide variety of stakeholders. With the tech industry constantly changing and evolving, I am excited about learning and using all the new technologies that will unfold.